File: 第1天/1.js
 
var log = function(logdata){
    console.log("Log", logdata)
    alert(logdata)
}

function sumAB(a, b){
	r = parseInt(a) + parseInt(b)
	return r
}


function decAB(a, b){
    r = parseInt(a) - parseInt(b)
    return r
} 



function mulAB(a, b){
	r = parseInt(a) * parseInt(b)
	return r
}


function divAB(a, b){
	if(parseInt(b) !== 0){
        r =parseInt(a) / parseInt(b)
    }else{
        r = False
    }
	return r
}
	

function run(a, ch, b){
    a = parseInt(a)
    b = parseInt(b)

	if(ch === "+"){
        r = sumAB(a, b)
    }else if(ch === "-"){
        r = decAB(a, b)
    }else if(ch === "*"){
        r = mulAB(a, b)
    }else if(ch === "/"){
        r = divAB(a, b)
    }else{
        r = False
    }	
	return r 
}
	

function main(){
	a = prompt("输入a： ")
	ch = prompt("输入运算符：")
	b = prompt("输入b:  ")
	r = run(a, ch, b)
	log(r)
}

main()

File: day001/1.py
# your code goes here
def log(*args, **kwargs):
	print("Log", *args, **kwargs)

def sumAB(a, b):
	r = int(a) + int(b)
	return r

def decAB(a, b): 
    r = int(a) - int(b)
    return r


def mulAB(a, b):
	r = int(a) * int(b)
	return r

def divAB(a, b):
	
	if int(b) != 0:
		r = int(a) / int(b)
	else:
		r = False
	return r

def run(a, ch, b):
	
	if ch == "+":
		r = sumAB(a, b)
	elif ch == "-":
		r = decAB(a, b)
	elif ch == "*":
		r = mulAB(a, b)
	elif ch == "/":
		r = divAB(a, b)
	else:
		r = False
	return r 

def main():
	a = input("输入a： ")
	ch = input("输入运算符：")
	b = input("输入b:  ")
	r = run(a, ch, b)
	log("结果是: ", r)
    

main()
    
   